model/group_request.go

```diff
@@ -70,6 +72,11 @@ func (p *PatchGroupRequest) Apply(group *Group) bool {
 		group.Version = *p.Version
 	}
 
+	if len(p.MattermostEnv) != 0 {
```

Comment:
With this check, are we unable to unset all the env vars if we want? It seems like that may be the case.

Reply:
No, it is the case that we are unable to unset all env vars with this check. This is a logical error, and instead on this line I should be checking `if p.MattermostEnv != nil`. This would mean that we can unset all env vars by providing an empty map, however an uninitialized map will be a no-op.